06_deploy_bootstrap_vm.sh

```diff
@@ -28,6 +28,9 @@ done
 IP=$(domain_net_ip ${INFRA_ID}-bootstrap baremetal)
 export API_VIP=$(dig +noall +answer "api.${CLUSTER_DOMAIN}" @$(network_ip baremetal) | awk '{print $NF}')
 echo "address=/api.${CLUSTER_DOMAIN}/${API_VIP}" | sudo tee /etc/NetworkManager/dnsmasq.d/openshift.conf
+export INGRESS_VIP=$(dig +noall +answer "ingress.${CLUSTER_DOMAIN}" @$(network_ip baremetal) | awk '{print $NF}')
```

Comment:
Here you are retrieving it from the libvirt hosts addresses, I suppose, so here it is fine to use ingress.